python/ray/tune/tuner.py

```diff
@@ -167,6 +167,14 @@ def restore(
         resume_unfinished: bool = True,
         resume_errored: bool = False,
         restart_errored: bool = False,
+        overwrite_trainable: Optional[
```

Comment:
nit: Name currently is an action which makes me intuitively think this takes a boolean value, wondering if we can change it to an object. Something like `overwritten_trainable` but with the connotation that it _will_ be the new trainable.

Reply:
I think `new_trainable` or even just `trainable` might actually be the clearest. I agree with `overwrite_trainable` seeming like a bool flag.